start.py

```diff
@@ -8,16 +10,42 @@
 from osfoffline.database import drop_db
 from osfoffline.gui.qt import OSFOfflineQT
 from osfoffline.utils.singleton import SingleInstance
+from osfoffline.settings import *
```

Comment:
Change this to `from osfoffline import settings`, please- that's more consistent with how it is imported elsewhere. `*` imports should be avoided if possible.